plugins/polio/api/campaigns/campaigns.py

```diff
@@ -947,10 +951,7 @@ def create_all_rounds_scopes_csv(self, request, **kwargs):
         org_units_list = []
         # loop on filtered rounds and make the org_units_list to be pushed in the csv file
         for round in rounds:
-            campaign = round.campaign
-            if not campaign:
-                continue
-            org_units_list += self.get_org_units_list(round, campaign)
+            org_units_list += self.get_org_units_list(round, round.campaign)
```

Comment:
@kemar Are you sure about this ?

Reply:
@mathvdh yup, this is now handled at line 931 with `exclude(Q(campaign__isnull=True)` and covered in tests.